packages/lib/src/toolbar/controls/InteractionHelp.module.css

```diff
@@ -0,0 +1,27 @@
+.root {
+  composes: root from '../OverflowMenu.module.css';
+}
+
+.menu {
+  composes: menu from '../OverflowMenu.module.css';
+}
+
+.menuList {
+  composes: menuList from '../OverflowMenu.module.css';
+}
+
+.menuList > li {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0.25rem 0.25rem;
+  white-space: nowrap;
+}
+
+.menuList > li > kbd {
+  background-color: var(--primary-light);
```

Comment:
Since it's a lib component, you should use a customizable CSS property starting with `--h5w` with a default value that doesn't rely on `@h5web/app`'s default theme properties. Something like `var(--h5w-interactionsHelp-shortcut-bgColor, lightgray)`, and in `App.module.css`: `--h5w-interactionsHelp-shortcut-bgColor: var(--primary-light)` (not forgetting to document it in Storybook of course).